src/mme/mme-s11-handler.c

```diff
@@ -342,6 +348,10 @@ void mme_s11_handle_create_session_response(
 
         ogs_debug("    ENB_S1U_TEID[%d] SGW_S1U_TEID[%d] PGW_S5U_TEID[%d]",
             bearer->enb_s1u_teid, bearer->sgw_s1u_teid, bearer->pgw_s5u_teid);
+
+        if (create_action == OGS_GTP_CREATE_IN_TRACKING_AREA_UPDATE &&
+            !OGS_FSM_CHECK(&bearer->sm, esm_state_active))
+            OGS_FSM_TRAN(&bearer->sm, esm_state_active);
```

Comment:
@acetcom let me know if you think there's a better way/place to do this transition, I couldn't find one.
See sequence diagram in 3GPP TS 23.401 D.3.6 for more information.

The point here is that we need to construct  the ue/session/bearer and move to the correct state from the information gained through Gn interface rather than the usual over S1AP/NAS/ESM. hence why this kind of new transitions appear in some code places.

I have so far tested the code paths using the following TTCN test which will be hopefully soon available and running nightly in https://jenkins.osmocom.org/jenkins/job/ttcn3-mme-test-ogs/:
https://gerrit.osmocom.org/c/osmo-ttcn3-hacks/+/35513